MainApp/LongWord/Program.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+
+namespace LongWord
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the text: ");
+            string text = Console.ReadLine();
+
+            string[] arrText = text.Split(' ');
+
+            string removeWord = removeLongestWord(arrText);
+            string swapWords = ChangeWordIndex(arrText);
+            int[] charactersCount = countCharacters(text);
+
+            Console.WriteLine($"Text without longest word: {removeWord}");
+            Console.WriteLine($"Swap longest and shortest words: {swapWords}");
+            Console.WriteLine($"Number of characters: {charactersCount[0]}");
+            Console.WriteLine($"Number of symbols: {charactersCount[1]}");
+            SortString(arrText);
+
+            Console.ReadLine();
+        }
+
+        static string removeLongestWord(string[] textArr)
+        {
+            int longWordIndex = 0;
+            int length = 0;
+
+            for (int i = 0; i < textArr.Length; i++)
+            {
+                if (textArr[i].Length > length)
+                {
+                    length = textArr[i].Length;
+                    longWordIndex = i;
+                }
+            }
+
+            textArr[longWordIndex] = "";
```

Comment:
I would suggest do not use hard-coded strings - use please `string.Empty` property